Gazebo quadrotor drivers fix

DanielYS opened the pull request:
Hello JdeRobot.

I've fixed an incompatibility problem with the quadrotor drivers for Gazebo. Now they are compatible with the changes made on the image slice interface. I've also added the interfaces needed to comunicate with them. The referee plugin for the cat-mouse test for Gazebo has been added too.

Best regards,

DanielYS

rocapal commented:
@DanielYS thanks for your help. So, I did this changes the last weekend. Can you update your local coopy of the repository and create a new pull request just with the new changes? 

Thanks

DanielYS commented:
@rocapal, I've updated the code and it runs well. I've removed the refereeplugin files from this pull request, so I will make another to upload them. The quadrotor driver's interfaces are still in the interfaces folder.

rocapal commented:
@DanielYS Thanks, I just checked and it compiles good. I'm going to merge this pull request.
